lib/instrumentation/queue.js

```diff
@@ -50,3 +55,9 @@ Queue.prototype._clear = function () {
   this._items = []
   this._timeout = null
 }
+
+// TODO: Check if there's an existing algorithm for this we can use instead
+function fuzzy (n, pct) {
+  var variance = n * pct * 2
```

Comment:
Untested, but this seems simpler:
`return (Math.random() * 2 - 1) * pct * n + n`